tests/snapshots/sample.modified.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
         "The perimeter of the rectangle is {} pixels.",
         perimeter(&rect1)
     );
-    println!(r#"This line contains invalid utf8:  "�"#;
```

Comment:
There are still some changes in this line (which is why the snapshots also show modifications in this line).

The `øˆ€€€` part is just the "interpolated" version of what comes out if you try to decode this as utf-8. You actually need to put the exact same bytes back in place (or tell me if you need any help).

Reply:
I might need a bit of help with that, sorry haha...

Reply:
Note that it can be pretty irritating when viewing the diffs in GitHub or some editors, as they will show the interpolated `øˆ€€€` string in both cases (if you actually put the `øˆ€€€` string in the file or if you put the invalid utf-8 sequence in the file)

Reply:
Ok, I'll look into this later today.